.idea/misc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
WARN! add .idea ignoring in .gitignore and AVOID pushing this files! DELETE them in your branch and push (this PR will be automatically updated! no need to cancel it, neither to do a new PR, KEEP this PR up!)